pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -226,6 +226,14 @@ def test_remove_nodes_from(self):
         self.assertRaises(ValueError, self.G1.get_cpds, 'diff')
         self.assertRaises(ValueError, self.G1.get_cpds, 'grade')
 
+    def test__eq__(self):
+        G4 = BayesianModel([('a', 'd'), ('b', 'd'),
+                                ('d', 'e'), ('b', 'c')])
+        G5 = BayesianModel([('a', 'd'), ('b', 'd'),
+                                ('d', 'e'), ('b', 'c')])
+        self.assertTrue(G4 == G5)
```

Comment:
Please add cpd to the `G4` and `G5` network and then test using `self.assertTrue(G4 == G5)`

Reply:
Could you please elaborate the reason for this?

Reply:
@Vaibhavdixit02  Currently `G4` and `G5` are just directed graphs and not PGM because they do not have factors associated with it. PGM allows us to represent joint probability distribution in an efficient way. A Bayesian Network consist of a directed graph and a Condition Probability Distribution associated with each random variable.Currently, G4 and G5 are just directed graphs and their is no CPD association. Using `self.assertTrue(G4 == G5)`, we want to check whether G4 Bayesian Model and G5 Bayesian Model are equal or not.

Reply:
That does seem correct but I think that `G4` and `G5` are just used as instantiations of `BayesianModel` objects here and their being PGM isn't really necessary? I mean the test case is just for comparing 2 `BayesianModel` object's equality? If it is necessary I will surely add your suggestion but I just want to be clear regarding its requirement. :)